news/_posts/2020-02-21-CFP-PrestaShop-Day-Paris-2020.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title:  "Call for paper for the PrestaShop Day Paris 2020"
+subtitle: "It's time to submit a talk to speak in front of the community"
+date:  2020-02-21 16:30:00
+authors: [AntoineThomas]
+icon: icon-board
+image: /assets/images/theme/meta-logo-build.png
+tags: [contribute, events, community]
+---
+
+
+Last year, a developer space was available at the PrestaShop Day Paris. This year, this space grows and will include an area for conferences and workshops dedicated to active contributors to the open source project.
+
+The event will take place in Paris, at "Palais Brongniart", June the 4th of 2020. More informations:  
+https://events.prestashop.com/prestashop-day-paris-2020/fr/
+
+## Thèmes
+
+Below is a list of themes for the call of papers that would be very appreciated this year.
+
+**Technique:**
+
+- FO or BO integration
+- Headless, PWA, API /. web-services
+- Information systems
+- Migration to PrestaShop
+- Performance, tests and quality
+
+**Project and community:**
+
+- Contribute to PrestaShop
+- Use of GitHub
+- Open source culture
+
+Do not hesitate to propose a topic around the PrestaShop project. All submissions will be studied.
+
+The target is to share and exchange knowledge, best practices and know how.
+
+## Formats
+
+Three conference formats are proposed in 2020:
+
+- 45min workshop
+- 30min talk
+- 5min lightning talk
+
+In order to respect the program, the speakers and the attendees at the show, no additional time will be allocated. Therefore, talks must be very well prepared.
```

Comment:
`In order to respect the program`
Please 😞 this is copy-paste from French to English.
Nobody says this in English.

Reply:
```suggestion
By respect for the program, the speakers and the attendees at the show, no additional time will be allocated. Therefore, talks must be very well prepared.
```